apiserver/facades/schema.json

```diff
@@ -29619,7 +29619,9 @@
         "Version": 11,
         "AvailableTo": [
             "controller-machine-agent",
-            "machine-agent"
+            "machine-agent",
+            "unit-agent",
+            "model-user"
```

Comment:
Ah, we're clearly changed some order here. There isn't much we can do about this. There is a proposal to fix this that people are happy with, but I don't have the time to do it.